docs/docs/css/extra.css

```
@@ -1,3 +1,23 @@
 h1, h2, h3 {
     margin-top: 77px;
 }
+
+#termynal {
+    height: 80ex !important; /* 40 lines of 2ex */
+    min-height: 80ex !important;
+    max-height: 80ex !important;
+    overflow: scroll !important;
+    font-size: 1.5ex !important;
+}
+
+[data-ty] {
+    line-height: 2ex !important; 
+}
```

Not sure why this is included for me locally by default, but fixes the single-line issue for the multi-select options.
```suggestion
    white-space: pre;
}
```